runtime/rococo/src/lib.rs

```diff
@@ -730,195 +1554,217 @@ parameter_types! {
+		ValidatorManager: validator_manager::{Pallet, Call, Storage, Event<T>} = 252,
 
-parameter_types! {
-	pub const MotionDuration: BlockNumber = 5;
-	pub const MaxProposals: u32 = 100;
-	pub const MaxMembers: u32 = 100;
+		// Sudo.
+		Sudo: pallet_sudo::{Pallet, Call, Storage, Event<T>, Config<T>} = 255,
+	}
 }
 
-impl pallet_collective::Config for Runtime {
-	type Origin = Origin;
-	type Proposal = Call;
-	type Event = Event;
-	type MotionDuration = MotionDuration;
-	type MaxProposals = MaxProposals;
-	type DefaultVote = pallet_collective::PrimeDefaultVote;
-	type MaxMembers = MaxMembers;
-	type WeightInfo = weights::pallet_collective::WeightInfo<Runtime>;
-}
+/// The address format for describing accounts.
+pub type Address = sp_runtime::MultiAddress<AccountId, ()>;
+/// Block header type as expected by this runtime.
+pub type Header = generic::Header<BlockNumber, BlakeTwo256>;
+/// Block type as expected by this runtime.
+pub type Block = generic::Block<Header, UncheckedExtrinsic>;
+/// A Block signed with a Justification
+pub type SignedBlock = generic::SignedBlock<Block>;
+/// `BlockId` type as expected by this runtime.
+pub type BlockId = generic::BlockId<Block>;
+/// The `SignedExtension` to the basic transaction logic.
+pub type SignedExtra = (
+	frame_system::CheckNonZeroSender<Runtime>,
+	frame_system::CheckSpecVersion<Runtime>,
+	frame_system::CheckTxVersion<Runtime>,
+	frame_system::CheckGenesis<Runtime>,
+	frame_system::CheckMortality<Runtime>,
+	frame_system::CheckNonce<Runtime>,
+	frame_system::CheckWeight<Runtime>,
+	pallet_transaction_payment::ChargeTransactionPayment<Runtime>,
+);
 
-impl pallet_membership::Config for Runtime {
-	type Event = Event;
-	type AddOrigin = EnsureRoot<AccountId>;
-	type RemoveOrigin = EnsureRoot<AccountId>;
-	type SwapOrigin = EnsureRoot<AccountId>;
-	type ResetOrigin = EnsureRoot<AccountId>;
-	type PrimeOrigin = EnsureRoot<AccountId>;
-	type MembershipInitialized = Collective;
-	type MembershipChanged = Collective;
-	type MaxMembers = MaxMembers;
-	type WeightInfo = weights::pallet_membership::WeightInfo<Runtime>;
-}
+/// Unchecked extrinsic type as expected by this runtime.
+pub type UncheckedExtrinsic = generic::UncheckedExtrinsic<Address, Call, Signature, SignedExtra>;
+/// Executive: handles dispatch to the various modules.
+pub type Executive = frame_executive::Executive<
+	Runtime,
+	Block,
+	frame_system::ChainContext<Runtime>,
+	Runtime,
+	AllPalletsWithSystem,
+>;
+/// The payload being signed in transactions.
+pub type SignedPayload = generic::SignedPayload<Call, SignedExtra>;
 
 parameter_types! {
-	// One storage item; key size is 32; value is size 4+4+16+32 bytes = 56 bytes.
-	pub const DepositBase: Balance = deposit(1, 88);
-	// Additional storage item size of 32 bytes.
-	pub const DepositFactor: Balance = deposit(0, 32);
-	pub const MaxSignatories: u16 = 100;
+	pub const ValidationUpgradeFrequency: BlockNumber = 2 * DAYS;
+	pub const ValidationUpgradeDelay: BlockNumber = 8 * HOURS;
```

Comment:
I think that `ValidationUpgradeFrequency` and `ValidationUpgradeDelay` are not being used anymore since we have `validation_upgrade_cooldown` and `validation_upgrade_delay` in [runtime parachains configuration](https://github.com/paritytech/polkadot/blob/master/runtime/parachains/src/configuration.rs#L77-L114)
Someone could confirm/deny this? Maybe @eskimor

Reply:
Right, actually those const are not being used anywhere. Removing them